<launch>
  <node pkg="pses_basis" type="uc_bridge" name="uc_bridge" output="screen">
    <param name="config_path" value="$(find pses_basis)/config/" />
    <param name="register_sensor_groups" value="true" />
    <param name="activate_motor_controller" value="true" />
    <param name="activate_steering_controller" value="true" />
    <param name="activate_kinect" value="true" />
    <param name="activate_us_sensors" value="true" />
    <param name="activate_daq" value="true" />
    <param name="reset_on_shutdown" value="true" />
  </node>
</launch>
